docs/core/diagnostics/dotnet-trace.md

```diff
@@ -3,17 +3,17 @@ title: dotnet-trace - .NET Core
 description: Installing and using the dotnet-trace command-line tool.
 author: sdmaclea
 ms.author: stmaclea
-ms.date: 10/14/2019
+ms.date: 10/28/2019
 ---
-# Trace for performance analysis utility (`dotnet-trace`)
+# dotnet-trace performance analysis utility
 
 **This article applies to:** .NET Core 3.0 SDK and later versions
 
-## Installing `dotnet-trace`
+## Install dotnet-trace
 
-To install the latest release version of the `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace), use the [dotnet tool install](../tools/dotnet-tool-install.md) command:
+Install `dotnet-trace` [NuGet package](https://www.nuget.org/packages/dotnet-trace) with the [dotnet tool install](../tools/dotnet-tool-install.md) command:
 
-```dotnetcli
+```console
```

Comment:
```suggestion
```dotnetcli
```

Reply:
This was incorrectly marked as resolved.

Reply:
It's been reverted back already.